lib/ttdrest/concerns/creative_types/flash.rb

```diff
@@ -26,6 +26,12 @@ def create_flash_creative(name, flash_content, clickthrough_url, landing_page_ur
             flash_data = flash_data.merge({"IsSecurable" => params[:is_securable]})
           end
 
+          if !params[:third_party_tracking_tags].nil?
```

Comment:
due to if this is nil, we don't want to send anything, because it will remove anything if we send an empty array.